recommenders/models/rbm/rbm.py

```diff
@@ -542,8 +444,8 @@ def init_metrics(self):
         """Initialize metrics"""
 
         if self.with_metrics:  # if true (default) returns evaluation metrics
-            self.Rmse = self.rmse(self.v_k)
-            self.Clacc = self.accuracy(self.v_k)
+            self.rmse = rmse(self.v, self.v_k)
+            self.accuracy = accuracy(self.v, self.v_k)
 
     def train_test_precision(self, xtst):
```

Comment:
This function `def train_test_precision(self, xtst):` is an example of not doing single responsibility and not doing DNRY. There should be one function for testing the precision of the train and test datasets, because intrinsecally they are the same. And this shouldn't be inside the fit function. The metrics are external, one thing is training an ML algo, and another thing is getting the evaluation of it. Does it make sense?

Also, why is this called precision if it is computed with the accuracy function, shouldn't be accuracy?

Reply:
I have removed the `train_test_precision` function altogether since the score calculated using this was not logged anywhere in the notebook. In the notebooks, precision and recall at K are printed using metrics functions from `utils`.